File: src/consumer_context.ts
/**
 * Contains information about the requesting user.
 */
export class ConsumerContext {
  /**
   * Contains the users identity. This is usually an auth token, created by an external authority.
   */
  public identity: string;
  /**
   * Contains the users language settings.
   */
  public internationalization?: string;
  /**
   * Contains the users locale settings.
   */
  public localization?: string;
}

File: src/dotnet/ConsumerContext.cs
namespace ProcessEngine.ConsumerAPI.Contracts
{
    /// <summary>
    /// Contains information about the requesting user.
    /// </summary>
    public class ConsumerContext
    {

        /// <summary>
        /// Contains the users identity. This is usually an auth token, created by an external authority.
        /// </summary>
        /// <value>The user identity token.</value>
        public string Identity { get; set; }
    }
}
